.github/workflows/ci-Instrumentation.Process.yml

```diff
@@ -0,0 +1,41 @@
+name: Build OpenTelemetry.Instrumentation.Process
+
+on:
+  pull_request:
+    branches: [ 'main*', 'instrumentation*' ]
+    paths:
+    - 'src/OpenTelemetry.Instrumentation.Process.Tests/**'
+    - 'test/OpenTelemetry.Instrumentation.Process.Tests/**'
```

Comment:
can you make some dummy change to this files, so that PR will trigger too

Reply:
Good catch, fixed.